examples/notebooks/05_market_comparison.ipynb

```diff
@@ -0,0 +1,363 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "OeeZDtIFmmhn"
+   },
+   "source": [
+    "## 2. Get ASSUME running\n",
+    "Here we just install the ASSUME core package via pip - just as we did in the other tutorials. In general the instructions for an installation can be found here: https://assume.readthedocs.io/en/latest/installation.html. All the required steps are executed here and since we are working in colab the generation of a venv is not necessary.  \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "m0DaRwFA7VgW",
+    "outputId": "5655adad-5b7a-4fe3-9067-6b502a06136b",
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install assume-framework"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "IIw_QIE3pY34"
+   },
+   "source": [
+    "And easy like this we have ASSUME installed. Now we can let it run. Please note though that we cannot use the functionalities tied to docker and, hence, cannot access the predefined dashboards in colab. For this please install docker and ASSUME on your personal machine.\n",
+    "\n",
+    "To run the examples, we still need some packages imports and configure a database server URI - you can adjust this if needed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "3eVM60Qx8SC0",
+    "outputId": "20434515-6e65-4d34-d44d-8c4529a46ece"
+   },
+   "outputs": [],
+   "source": [
+    "from functools import partial\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "from sqlalchemy import create_engine\n",
+    "\n",
+    "from assume import World\n",
+    "from assume.scenario.loader_csv import load_scenario_folder\n",
+    "\n",
+    "# make sure that you have a database server up and running\n",
+    "DB_URI = \"postgresql://assume:assume@localhost:5432/assume\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can run the first scenario which does only include the energy-only market, we call this \"eom_case\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "world = World(database_uri=DB_URI)\n",
+    "load_scenario_folder(\n",
+    "    world,\n",
+    "    inputs_path=\"examples/inputs\",\n",
+    "    scenario=\"example_02c\",\n",
+    "    study_case=\"eom_case\",\n",
+    ")\n",
+    "world.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can run the second case which also includes the long term market as can be seen in the config.yaml of it:"
```

Comment:
Here also maybe just printing it or showing it as a dict? Easier for users to compare directly with the previous cell